model/junit_test_result.go

```diff
@@ -15,7 +15,8 @@ type JunitTestResult struct {
 	Start time.Time `datastore:",index"`
 	End   time.Time `datastore:",index"`
 
-	Results UnitTestResults `datastore:",noindex"`
+	Results  UnitTestResults `datastore:",noindex"`
+	Endpoint string
```

Comment:
What is this endpoint used for again? To render results afterwards? Is it used currently?